src/test/scala/org/zalando/zhewbacca/IAMClientSpec.scala

```diff
@@ -3,21 +3,23 @@ package org.zalando.zhewbacca
 import akka.actor.ActorSystem
 import org.specs2.mutable.Specification
 import org.zalando.zhewbacca.metrics.NoOpPlugableMetrics
-import play.api.http.Port
+import play.api.http.{DefaultFileMimeTypes, FileMimeTypesConfiguration, Port}
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.libs.ws.WSClient
-import play.api.mvc.{Action, Handler, Result, Results}
+import play.api.mvc._
 import play.api.test.WsTestClient
 import play.api.{Application, Configuration, Mode}
 import play.core.server.Server
 
 import scala.concurrent.duration._
 import scala.concurrent.{Await, ExecutionContext}
 
-// @see https://www.playframework.com/documentation/2.5.x/ScalaTestingWebServiceClients
+// @see https://www.playframework.com/documentation/2.6.x/ScalaTestingWebServiceClients
```

Comment:
The remaining deprecation warnings are caused by using ```Action``` for route definitions here. The proper way to fix this might be using ```Server.withRouterFromComponents()``` as described in [ScalaTestingWebServiceClients](https://www.playframework.com/documentation/2.6.x/ScalaTestingWebServiceClients) but I haven't had the time to try this.